docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -127,6 +127,42 @@ supports the following features:
   :ref:`iceberg-materialized-views`
 * :ref:`sql-views-management`
 
+.. _iceberg-alter-table-execute-optimize:
+
+ALTER TABLE EXECUTE OPTIMIZE
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector offers :ref:`ALTER TABLE EXECUTE <alter-table-execute>`
+``OPTIMIZE`` command for collapsing files which correspond to the
```

Comment:
@findepi please do provide feedback in case that I'm wrong in the assumption that `OPTIMIZE` touches only the files which correspond to the current snapshot of the table.

Reply:
1. yes it reads off of current snapshot
2. talking about snapshots here is not necessary. Do you specify that UPDATE or SELECT operate on current snapshot? worse, it can be misunderstood, as if OPTIMIZE was replacing current snapshot, or rewriting the files in situ